pkg/module_manager/module_hook.go

```diff
@@ -146,7 +146,7 @@ func (h *ModuleHook) Run(bindingType BindingType, context []BindingContext, logL
 	})
 
 	logEntry := log.WithFields(utils.LabelsToLogFields(logLabels))
-	logEntry.Info("Module hook start")
+	logEntry.Infof("Module hook start %s/%s\n", h.Module.Name, h.Name)
```

Comment:
Do we need names in the 'msg' filed, as `h.Module.Name` and `h.Name` are already in labels?

```suggestion
	logEntry.Infof("Module hook start %s/%s", h.Module.Name, h.Name)
```